app/code/community/Ebizmarts/MailChimp/Model/Mysql4/Interestgroup.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * mc-magento Magento Component
+ *
+ * @category  Ebizmarts
+ * @package   mc-magento
+ * @author    Ebizmarts Team <info@ebizmarts.com>
+ * @copyright Ebizmarts (http://ebizmarts.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ * @date:     6/9/16 4:46 PM
+ * @file:     Interestgroup.php
+ */
+class Ebizmarts_MailChimp_Model_Mysql4_Interestgroup extends Mage_Core_Model_Mysql4_Abstract
+{
+
+    /**
+     * Initialize
+     *
+     * @return void
+     */
+    public function _construct()
+    {
+        $this->_init('mailchimp/interestgroup', 'id');
+    }
+
+    public function getByRelatedIdStoreId($customerId = 0, $subscriberId = 0, $storeId)
+    {
+        $read = $this->_getReadAdapter();
+        $select = $read->select()
+            ->from($this->getMainTable())
+            ->where('store_id = ?', $storeId)
+            ->where('(' . $read->quoteInto('customer_id = ?', $customerId) . ' OR ' . $read->quoteInto('subscriber_id = ?', $subscriberId) . ')');
```

Comment:
can we user `orWhere` instead of where? https://github.com/OpenMage/magento-lts/blob/1.9.3.x/app/code/core/Zend/Db/Select.php#L496-L513

Reply:
In the doc: https://framework.zend.com/manual/1.10/en/zend.db.select.html
Example #20 Example of parenthesizing Boolean expressions uses the or in the same was as here.